.gitignore

```diff
@@ -3,6 +3,5 @@ Paper.aux
 Paper.bbl
 Paper.blg
 Paper.log
-Paper.pdf
```

Comment:
Let's not put the PDF itself into the repository

Reply:
What's the reason for that? We can then update the PDF every time you do a new local build and push, and link to it. It will also be accessible directly from the repo.

Reply:
I guess a problem is that if you clone the repo, and build, then you'll have two PDFs. So you'd have to delete the original PDF before building and pushing the modified changes. But I don't see this as a big deal, and I still think it is a net benefit to include the PDF in the repo.

Reply:
There is an automated process that builds the PDF and uploads it to the link mentioned in the README

Reply:
OK, makes sense.